javaCalc/src/Init.java

```diff
@@ -0,0 +1,14 @@
+import java.awt.*;
+import java.awt.event.*;
+import javax.swing.*;
+
+public class Init {

```

Comment:
Change this name pls, it doesn't describe at all what it does. ButtonFactory or something similar would be better